src/Diagnostics/Test/Utilities/DiagnosticAnalyzerTestBase.cs

```diff
@@ -53,7 +53,7 @@ protected static DiagnosticResult GetGlobalResult(DiagnosticDescriptor rule, par
             {
                 Id = rule.Id,
                 Severity = rule.DefaultSeverity,
-                Message = rule.MessageFormat.ToString()
+                Message = string.Format(rule.MessageFormat.ToString(), messageArguments)
```

Comment:
The problem here is I need to create a `DiagnosticResult` with the full error message text (replaced the holes in the format string).  Otherwise it won't line up with the actual result.  

I checked for other uses of `GetGlobalResult` in the code base and there were none I could find.